arch/arm/configs/tsimx6ul_defconfig

```diff
@@ -857,26 +714,19 @@ CONFIG_DRM_IMX=y
 CONFIG_DRM_IMX_PARALLEL_DISPLAY=y
 CONFIG_DRM_MXSFB=y
 CONFIG_FB_MODE_HELPERS=y
+# CONFIG_FB_MX3 is not set
 CONFIG_FB_ST7565P=m
-CONFIG_DRM_FBDEV_EMULATION=y
 CONFIG_LCD_CLASS_DEVICE=y
+CONFIG_LCD_PLATFORM=y
 CONFIG_BACKLIGHT_CLASS_DEVICE=y
-CONFIG_BACKLIGHT_PWM=m
-CONFIG_BACKLIGHT_GPIO=m
-CONFIG_SOUND=y
-CONFIG_SND=y
-# CONFIG_SND_ARM is not set
-# CONFIG_SND_SPI is not set
-CONFIG_SND_USB_AUDIO=m
-CONFIG_HID_MULTITOUCH=y
+CONFIG_BACKLIGHT_PWM=y
+CONFIG_BACKLIGHT_GPIO=y
+CONFIG_HID_MULTITOUCH=m
+CONFIG_USB_HID=m
 CONFIG_USB=y
 CONFIG_USB_ANNOUNCE_NEW_DEVICES=y
-CONFIG_USB_OTG=y
-CONFIG_USB_OTG_FSM=m
-CONFIG_USB_LEDS_TRIGGER_USBPORT=m
```

Comment:
6ul absolutely supports OTG, keep this in.

Reply:
Fixed